packages/toolbox-core/src/toolbox_core/utils.py

```diff
@@ -45,41 +45,64 @@ def create_func_docstring(description: str, params: Sequence[ParameterSchema]) -
 
 
 def identify_required_authn_params(
```

Comment:
nit: probably time to rename this "idenfity_auth_requirements" given the multiple return types